monai/transforms/post/contour.py

```diff
@@ -0,0 +1,46 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+import torch.nn as nn
+import numpy as np
+
+class Contour:
+    """
+    Return the contour of the organ in a mask image that only composes of 0 and 255 in gray value, 
```

Comment:
should suppose values of [0, 1] instead of [0, 255].

Reply:
I prefer [0, 255], because they respectively represent the gray value of black and white. and the form accords with image binarization.

Reply:
This post transform is used for `model output` instead of normal 2D images, should not be 255. And even for normal mages, `255` is not commonly used in medical images.
So I think you can suppose input data is within [0, 1] values.

Reply:
ok, get your point, updated the code, please help review again.